knpu/authorization-code.md

```diff
@@ -124,12 +129,14 @@ a little further down::
         // equivalent to $_GET['code']
         $code = $request->get('code');
 
-        die('Implement this in CoopOAuthController::receiveAuthorizationCode');
+        die('Implement this in CoopOAuthController:receiveAuthorizationCode');
```

Comment:
Hmm, weird thing, I specified that only replace if it ends with break line
for some reason I missed this one, nice catch btw ;)